### ISSUE TITLE
Undefined rule scss/declaration-nested-properties

### ISSUE THREAD

**Issue by dustinhorton** [kristerkari/stylelint-scss#90]
Thanks for this project.

I'm using `stylelint@7.6.0` & `stylelint-scss@1.3.4`. When I enable [declaration-nested-properties](https://github.com/kristerkari/stylelint-scss/blob/master/src/rules/declaration-nested-properties/README.md), via:

```js
'scss/declaration-nested-properties': 'never'
```

I get an error about it being undefined:

```console
Error: Undefined rule scss/declaration-nested-properties
```

Let me know if I can provide any more info.

**Comment by dustinhorton** [comment]
Ah, coming next release it would seem: https://github.com/kristerkari/stylelint-scss/blob/9100028152dc6ce7066c18ec104ebaca45268fac/CHANGELOG.md#head

**Comment by dryoma** [comment]
@dustinhorton  The rules are now available in 1.4.1. Took us quite long :(
